app/gui/src/project-view/providers/popoverRoot.ts

```diff
@@ -10,5 +10,5 @@ import { type Ref } from 'vue'
  */
 export const [providePopoverRoot, usePopoverRoot] = createContextStore(
   'popoverRoot',
-  identity<Ref<HTMLElement | undefined | null>>,
+  identity<ToValue<HTMLElement | undefined | null>>,
```

Comment:
I think usage of this API is simpler if the value it produces is a more specific type than `ToValue`--it looks like `Readonly<Ref<..>>` would suit the consumers and producer.